api/tests/unit/features/test_unit_feature_external_resources_views.py

```diff
@@ -41,20 +40,31 @@ def json(self):
 def test_create_feature_external_resource(
     admin_client_new: APIClient,
     feature_with_value: Feature,
+    environment: Environment,
     project: Project,
     github_configuration: GithubConfiguration,
     github_repository: GithubRepository,
-    mocker,
+    mocker: MockerFixture,
 ) -> None:
     # Given
     mock_generate_token = mocker.patch(
         "integrations.github.github.generate_token",
     )
+
     mock_generate_token.return_value = "mocked_token"
     github_request_mock = mocker.patch(
         "requests.post", side_effect=mocked_requests_post
     )
-    datetime_now = datetime.datetime.now()
+
+    expected_comment_body = (
+        "**Flagsmith feature linked:** `feature_with_value`\n"
+        + "Default Values:\n"
+        + "| Environment | Enabled | Value | Last Updated (UTC) |\n"
+        + "| :--- | :----- | :------ | :------ |\n"
+        + f"| [Test Environment](https://example.com/project/{project.id}/"
+        + f"environment/{environment.api_key}/features?feature={feature_with_value.id}&tab=value) "
+        + "| ❌ Disabled | `value` | 2024-01-01 00:00:00 |\n"
+    )
```

Comment:
This is nice! nit: Python supports multiline strings so you could do this:

```suggestion
    expected_comment_body = (
        "**Flagsmith feature linked:** `feature_with_value`\n"
        "Default Values:\n"
        "| Environment | Enabled | Value | Last Updated (UTC) |\n"
        "| :--- | :----- | :------ | :------ |\n"
        f"| [Test Environment](https://example.com/project/{project.id}/"
        f"environment/{environment.api_key}/features?feature={feature_with_value.id}&tab=value) "
        "| ❌ Disabled | `value` | 2024-01-01 00:00:00 |\n"
    )
```

Looks a bit nicer this way, IMO.